### ISSUE TITLE
Change `swagger_generated` name

### ISSUE THREAD

**Issue by wires** [TinkerTravel/partner-api-client-python#3]
`swagger_generated` is the name of the package containing the generated python bindings to the rest API, this should be changed

**Comment by NinoDoko** [comment]
Yea, that's a small change. `tinker_rest_api` sound good?

**Comment by wires** [comment]
Yes, I first thought maybe `tinker_parter_api` was better but then we already used that, but after a bit of deliberating I think `tinker_rest_api` is actually better, since it follows the REST API semantics. anyway :-) Sorry for the late response

**Comment by NinoDoko** [comment]
Well, I made a new branch and I changed the name as well as added some error handling, but I don't have permission to push to the repo. I wanted to make a pull request so you can review the code before merging it with master.
